src/Auth/Login.tsx

```diff
@@ -0,0 +1,77 @@
+import { Form, Input, Button, Row, Col } from 'antd';
+import { EyeInvisibleOutlined, EyeTwoTone } from '@ant-design/icons';
+import { Link, useHistory } from 'react-router-dom';
+import { useAuth } from '../contexts/Auth';
+
+interface LoginProps {
+  email: string;
+  password: string;
+}
+export const Login = () => {
+  const [form] = Form.useForm();
+
+  // Get signUp function from the auth context
+  const { signIn } = useAuth();
+  const history = useHistory();
+
+  async function handleSumit(e: LoginProps) {
```

Comment:
You can wrap this on a `useCallback` hook. See https://dmitripavlutin.com/dont-overuse-react-usecallback/ for why this is important